src/main/java/appeng/items/tools/powered/ToolWirelessTerminal.java

```diff
@@ -19,14 +19,27 @@
 package appeng.items.tools.powered;
 
 
+import java.awt.*;
 import java.util.EnumSet;
 import java.util.List;
 
+import appeng.api.util.AEColor;
+import appeng.client.render.BaseBlockRender;
+import appeng.client.render.blocks.RenderBlockSkyCompass;
+import appeng.client.render.items.ToolBiometricCardRender;
+import appeng.client.render.items.ToolWirelessTerminalRender;
+import appeng.core.localization.PlayerMessages;
+import appeng.recipes.game.ShapelessRecipe;
 import com.google.common.base.Optional;
 
+import net.minecraft.client.renderer.texture.IIconRegister;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.item.ItemStack;
+import net.minecraft.item.crafting.FurnaceRecipes;
+import net.minecraft.item.crafting.ShapedRecipes;
+import net.minecraft.item.crafting.ShapelessRecipes;
 import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.IIcon;
 import net.minecraft.util.StatCollector;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.util.StatCollector'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)